### ISSUE TITLE
mac ERROR - Fatal: No such file or directory; did you install phantomjs?

### ISSUE THREAD

**Issue by txyzwh** [tebelorg/RPA-Python#181]
1. i have install phantomjs

> `
(py36) taoxiyan@taoxiyandeMacBook-Pro ~ % brew cask install phantomjs
Warning: Cask 'phantomjs' is already installed.

To re-install phantomjs, run:
  brew reinstall phantomjs
(py36) taoxiyan@taoxiyandeMacBook-Pro ~ % brew reinstall phantomjs
==> Caveats
phantomjs has been officially discontinued upstream.
It may stop working correctly (or at all) in recent versions of macOS.

==> Downloading https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-
Already downloaded: /Users/taoxiyan/Library/Caches/Homebrew/downloads/f8cdfacce271a45f2b21ff12f7cd783d941450ed6d34e5c20eb20abd42f481c0--phantomjs-2.1.1-macosx.zip
==> Verifying SHA-256 checksum for Cask 'phantomjs'.
==> Uninstalling Cask phantomjs
==> Unlinking Binary '/usr/local/bin/phantomjs'.
==> Purging files for version 2.1.1 of Cask phantomjs
==> Installing Cask phantomjs
==> Linking Binary 'phantomjs' to '/usr/local/bin/phantomjs'.
🍺  phantomjs was successfully installed!`

2. when i install the tagui i follow the guide : https://tagui.readthedocs.io/en/latest/setup.html
Download TagUI v6.0.0 (macOS, Linux).
Unzip the contents to your desktop on macOS, or /home/your_userid on Linux.
Install OpenJDK (macOS, Linux).
Install Chrome.
Open Terminal.
Copy, paste and run these commands, replacing your_tagui_path as needed:
`ln -sf /your_tagui_path/tagui/src/tagui /usr/local/bin/tagui`

`(base) taoxiyan@taoxiyandeMacBook-Pro ~ % ln -sf /Users/taoxiyan/tagui/src/tagui /usr/local/bin/tagui
(base) taoxiyan@taoxiyandeMacBook-Pro ~ % `

3. can not run the sample provided
`(base) taoxiyan@taoxiyandeMacBook-Pro ~ % tagui /Users/taoxiyan/tagui/flows/samples/1_google.tag
zsh: permission denied: tagui`

4. can not init the tagui
`(base) taoxiyan@taoxiyandeMacBook-Pro ~ % python 
Python 3.7.6 (default, Jan  8 2020, 13:42:34) 
[Clang 4.0.1 (tags/RELEASE_401/final)] :: Anaconda, Inc. on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import tagui as t
>>> t.init()
[RPA][ERROR] - following happens when starting TagUI...

ERROR - Fatal: No such file or directory; did you install phantomjs?

False`

**Comment by txyzwh** [comment]
`(base) taoxiyan@taoxiyandeMacBook-Pro ~ % brew list 
bash-completion		graphviz		netpbm
cairo			gts			node
fontconfig		harfbuzz		pango
freetype		icu4c			pcre
fribidi			jasper			pixman
gd			jpeg			pkg-config
gdbm			libffi			python@3.8
gettext			libpng			readline
glib			libtiff			sqlite
gobject-introspection	libtool			webp
graphite2		lzo			xz`

**Comment by txyzwh** [comment]
`(base) taoxiyan@taoxiyandeMacBook-Pro ~ % brew list ‘
bash-completion		graphviz		netpbm
cairo			gts			node
fontconfig		harfbuzz		pango
freetype		icu4c			pcre
fribidi			jasper			pixman
gd			jpeg			pkg-config
gdbm			libffi			python@3.8
gettext			libpng			readline
glib			libtiff			sqlite
gobject-introspection	libtool			webp
graphite2		lzo			xz

**Comment by ck81** [comment]
Hi @txyzwh. CK here.

You shouldn't need to manually install phantomjs.

The guide you have used: https://tagui.readthedocs.io/en/latest/setup.html
is for TagUI, not TagUI-Python.

Try to use the guide here: [Set up TagUI-Python on Mac](https://yuchenkuang.com/TagUI-Python-Setup/setup-tagui-python-on-mac.php)

Follow the guide there step-by-step.  If you bump into any problem, please let me know which part you are stuck and what's the error message that you see on the screen.

**Comment by kensoh** [comment]
Hi RPA for Python users, 🅗🅐🅟🅟🅨 ②⓪②①!

It has been many months since I last worked on this personal side project. Just want to shout out that this project is very much alive, and I hope to look into the 30+ issues raised since July (and bug-fixes if required) soon when I get a moment again.

Below are details if you are interested, why I'm away past few months and why I believe my darkest days are over 🍀🤞🏻🙏🏻

---

Since May last year, I've stopped all work to juggle with [my mum's critical illness](https://github.com/tebelorg/RPA-Python/issues/144), taking care of my toddler, and doing housework chores like cleaning and cooking. TBH, my wife and I felt that taking care of a baby and giving her the best nutrition (nursing and home-cooked organic food) and care requires at least 1.5 full-time headcount. There is no life to speak of, other than trying to stay afloat with baby care needs, and keep finding alternative ways to treat my mum, where there're some wins.

Eg importing soy-based tube feeds from Netherlands / Germany / Australia --> after switching away from milk-based tube feeds for 2 weeks she stops requiring dialysis. She started requiring 3x a week dialysis for more than 3 months, after admitting to the hospital. Also, applying herbs like mashed garlic or diced onions to her feet somehow consistently improve her breathing and lung function. Have to keep trying alternative healing methods when her recovery keeps hitting plateau in hospital.

She has been discharged from the hospital a month ago and recovering at home, though very slowly. A longterm medication seems to be one of the contributing causes for her condition. After stopping it, there seems to be improvement to her breathing capacity. She is still bed-bound, but over the course of the year I hope she'll regain strength to move about, and hopefully lungs to recover sufficiently to not require oxygen tank support through a tracheal mask (a hole through her throat).

---

Since November, I've gone back to work at [AI Singapore](https://www.aisingapore.org), a government-funded programme to accelerate AI in Singapore. I'm with Product Engineering team, focusing on RPA and [TagUI (open-source RPA tool)](https://github.com/kelaberetiv/TagUI), which RPA for Python is a wrapper for. There are pretty interesting stuffs on [TagUI 2021 roadmap](https://www.linkedin.com/posts/kensoh_tagui-rpa-v614-release-just-made-it-in-time-activity-6747502665758072833-6YUu). 2020 was the year which my family was going to move to the UK and work, after I receive its tier 1 exceptional tech talent visa. Then everything changed with the virus outbreak, and my mum's illness. The best thing that happened to me in 2020 was being able to go back to work at AI Singapore - I'm grateful.

I've just hired someone to help with housework, cooking and supporting my wife with baby care. I hope this new arrangement will allow me to deliver on my work at AI Singapore, and soon be able to reply countless friends' messages which circumstances leave me with no choice but go missing for months, and then look into the issues raised here!

Eg of the cost of best-in-class nutrition standard researched and set by wife. Baby just felt restless and broke her plate. Even Corelle's break-resistant plate breaks. Why not use plastic plate? Cuz plastic can leech toxins to the food on it. Why not use Silicon plate? Cuz there is no white silicon materials for plate. Why white? Cuz that helps to improve visibility of the food instead of contrasting against some pink colour or some other colours. I would be spending much time clearing up the mess of 70 over pieces of plate fragments, while baby gets nursed to calm down, if not for our newfound helper.

That's why I feel like my darkest days are over and now seeing glimpses of light at the end of the tunnel.

---

My vision for RPA for Python remains - I would like it to be the ***go-to Python package for RPA***.

A best-in-class tool in Python users toolbox which you combine with other cool packages to build amazing stuffs.
